src/enums/componentsReviewCertify.ts

```diff
@@ -10,6 +10,7 @@ export enum ComponentsReviewCertify {
   'completing-party-section',
   'transactional-folio-number-section',
   'detail-section',
+  'special-resolution-confirm', // to focus on this component before certify
```

Comment:
Is this comment specific to this component, or does it apply to all components? Is this comment needed?

Reply:
I notice that it mentioned on top